App/main.js

```diff
@@ -46,6 +46,13 @@ const ui = {
 
   submitTwitterScoreDiv: $('#submit_twitter_score_div'),
   submitTwitterScoreLink: $('#submit_twitter_score_link'),
+  submitRedditScoreDiv: $('#submit_reddit_score_div'),
+  submitRedditScoreLink: $('#submit_reddit_score_link'),
+  submitRedditScoreSubreddit: $('#submit_reddit_score_subreddit'),
+
+  redditOpenModal: $('#redditOpen-bar'),
```

Comment:
Small nitpick - and really I am also to blame here if you look at lines 47/48 (twitter ids) - we probably should use the same casing as seemingly all other vars here (lower case, hyphenated) as opposed to mixing camelCase + underscores/etc)

Reply:
Issue should be resolved.
Also fixed the twitter ids